### DISCUSSION TITLE
Is it possible to hide the original volume slider of the monitor?

### DISCUSSION THREAD

**Discussion by carlosoonline** [MonitorControl/MonitorControl#1111] [General]
I have an LG 27UK580 monitor and a MacMini M1, when I change the volume from the keyboard (apple magic keyboard), the MacOS UI appears, which is perfect, I don't want the original monitor interface to appear. Only the MacOS UI should appear.

![749441A4-059E-4B81-B9C9-4A6DA674D250](https://user-images.githubusercontent.com/8492679/178089606-7e6c0c8b-0a33-45e9-abae-31d3e92b04ea.jpg)

**Comment by waydabber** [comment]
No, it is not possible to suppress the LG's volume OSD. There were attempts in the past but they did not work properly.
